llvm/utils/git/code-lint-helper.py

```diff
@@ -291,39 +390,58 @@ def run_linter(changed_files: List[str], args: LintArgs) -> tuple[bool, Optional
         default="clang-tidy",
         help="Path to clang-tidy binary",
     )
+    parser.add_argument(
+        "--doc8-binary",
+        type=str,
+        default="doc8",
+        help="Path to doc8 binary",
+    )
     parser.add_argument(
         "--verbose", action="store_true", default=True, help="Verbose output"
     )
 
     parsed_args = parser.parse_args()
     args = LintArgs(parsed_args)
 
-    changed_files = []
-    if args.changed_files:
-        changed_files = args.changed_files.split(",")
+    overall_success = True
+    failed_linters = []
+    all_comments = []
```

Comment:
Missing type annotation. `PyLance` in Visual Studio Code is great tool to detect such problems.